bootloader/storage/sdmmc.c

```diff
@@ -707,15 +707,10 @@ int _sd_storage_get_scr(sdmmc_storage_t *storage, u8 *buf)
 	u32 tmp = 0;
 	sdmmc_get_rsp(storage->sdmmc, &tmp, 4, SDMMC_RSP_TYPE_1);
     //Prepare buffer for unstuff_bits
-	for (int i = 0; i < 8; i+=4)
-	{
-		storage->raw_scr[i + 3] = buf[i];
-		storage->raw_scr[i + 2] = buf[i + 1];
-		storage->raw_scr[i + 1] = buf[i + 2];
-		storage->raw_scr[i]     = buf[i + 3];
-	}
+	for (int i = 0; i < 2; i++)
+		storage->raw_scr[i] = __builtin_bswap32(*(u32 *)&buf[i * sizeof(u32)]);
```

Comment:
We have our independent byte_swap_32(num) for this.

Reply:
You should change byte_swap_32 to just be __builtin_bswap32